app/src/main/java/com/nextcloud/talk/utils/database/user/CapabilitiesUtilNew.java

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
No new files in Java! :coffee: 

Additionally the same note about deprecation duplication applies here. Change methods in the old CapabilityUtils to call the ones in this one after converting the `UserEntity` to a `User`.

Reply:
That way you migrate all call-path to the methods in the old utils class at once which means you change executed code throughout the whole app, which implies testing everything. If you work with a adapted copy and migrate it piece by piece you can then remove the old methods when all references have been gone. This way you have more, fine-grained control over what you change, so I would suggest not do this.

Reply:
> If you work with a adapted copy and migrate it piece by piece you can then remove the old methods when all references have been gone.

Fair enough, but with the current approach we run into risk of inconsistencies if we forget to modify both classes at the same time. Your call